src/components/MovieCard/moviecard.js

```diff
@@ -37,7 +37,8 @@ const MovieCard = ({ title, description, year, onClick }) => {
   return (
     <div className="movie-card">
       <div className="image-container">
-        <img onClick={onClick} className="movie-image" src={movieimage} alt={title} />
+        <img onClick={onClick} className="movie-image" src={image}
+             onError={(e)=>{e.target.onerror = null; e.target.src=defaultmovieimage}}/>
```

Comment:
this logic can be extracted from the component and placed along

Reply:
`e.target.onerror = null;`
I think that one is not really necessary and also I believe handler will be set again after rerender

Reply:
removed onError handler